public/index.php

```diff
@@ -9,7 +9,12 @@
   </li>
   <li>
     <a href="discussions.php"><strong>View</strong></a> - list all discussions
+  </li> 
+  <li>
+    <a href="InstructorsList.php"><strong>View</strong></a> - Instructors
   </li>
```

Comment:
maybe we should stick to naming the files in lowercase

Reply:
done